packages/odyssey-storybook/.storybook/preview-head.html

```diff
@@ -27,4 +27,16 @@
   .sbdocs .sbdocs-a {
     color: #1662dd;
   }
+
+  .sbdocs .sb-dont::after {
```

Comment:
We didn't end up needing this decorator for the `Banner` docs, so it's not in play in this PR. We will want it going forward, though. Happy to merge or remove as folks see fit.

Reply:
minor, but given the choice I think I'd prefer to see this merge in when we need it as that will help with comprehension and if necessary, a future revert etc.